Packs/Base/ReleaseNotes/1_18_17.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### CreateIndicatorRelationship
+- Removed *entity_b_type* argument from required 
+- Fixed an issue where ***CreateIndicatorRelationship*** command would fail when using *entity_b_query* argument without *entity_b_type* argument
```

Comment:
```suggestion
- Fixed an issue where the ***CreateIndicatorRelationship*** command would fail when using the *entity_b_query* argument without providing the *entity_b_type* argument.
```